MovieChecker/MovieChecker/RottenTomatoesHelperMethods.h

```diff
@@ -0,0 +1,16 @@
+//
+//  RottenTomatoesHelperMethods.h
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/22/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface RottenTomatoesHelperMethods : NSObject
+
++ (NSURL *)boxOfficeURL;
++ (NSArray *)interpretBoxOfficeMoviesFromData:(NSData *)data withkey:(NSString *)key;
```

Comment:
Is there a reason for these to exist outside of `MovieList`?  If not, consider moving them inside that class.